test/testdata/trustroot/e2e/bring-your-own-keys.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+apiVersion: policy.sigstore.dev/v1alpha1
+kind: TrustRoot
+metadata:
+  name: my-sigstore-keys
+spec:
+  sigstoreKeys:
+    certificateAuthorities:
+    - subject:
+        organization: fulcio-organization
+        commonName: fulcio-common-name
+      uri: https://fulcio.fulcio-system.svc
+      certChain: |-
+        FULCIO_CERT_CHAIN
+    tLogs:
```

Comment:
I know we added this before. What about using `tlogs` and `ctlogs` as the name here ?

Reply:
If we do the rename (which I don't have strong feelings about), I'd rather do those renames as part of a PR that only does that, you ok with that?